docs/excel/custom-functions-overview.md

```diff
@@ -10,6 +10,9 @@ ms.localizationpriority: high
 
 Custom functions enable developers to add new functions to Excel by defining those functions in JavaScript as part of an add-in. Users within Excel can access custom functions just as they would any native function in Excel, such as `SUM()`.
 
+> [!NOTE]
+> **Custom function** is a general term that is interchangeable with **user-defined function**. Both terms apply to VBA, COM, and Office.js add-ins. The Office Add-ins documentation only uses the term **custom function** when referring to custom functions that use Office.js APIs.
```

Comment:
I think "only" makes this seem like we have other terms for the broader concept. Also, I believe the term "Office JavaScript APIs" is more common than "Office.js APIs" in our docs.

```suggestion
> **Custom function** is a general term that is interchangeable with **user-defined function**. Both terms apply to VBA, COM, and Office.js add-ins. The Office Add-ins documentation uses the term **custom function** when referring to custom functions that use Office JavaScript APIs.
```